pywbem/cim_operations.py

```diff
@@ -486,6 +487,31 @@ class WBEMConnection(object):  # pylint: disable=too-many-instance-attributes
     the :attr:`last_raw_request` and :attr:`last_raw_reply` attributes
     of the connection object.
 
+    WBEMConnection objects can record the results of
+    :class:`~pywbem.WBEMConnection` methods that interact with
+    a WBEM server using either the :class:`~pywbem.LogOperationRecorder` (uses
+    python logging facitlity to create logs for these methods and http
+    interactions) or the :class:`~pywbem.TestClientRecorder`( creates a yaml
+    file of methods     and http for tests)..
+
+    Before pywbem version 0.11.0, there was only a single operation recorder and
+    activating/deactivating the recorder was a simple matter of setting the
+    value of an instance of the :class:`~pywbem.TestClientRecorder` into the
+    WBEMConnection.operation_recorder attribute.
+    Setting the value of :class:`TestClientRecorder` into this
+    attribute activated and enabled the recorder.  Setting it to None
+    deactivated the recorder.
+
+    Starting with pywbem version 0.11.0, pywbem added a second recorder and the
+    requirement     that both recorders operate simultaneously. Thus, if both
+    are enabled and active, operation data can be both recorded to a yaml file
+    and to the log simultaneously.
+
+    The method to activate has therefore change. To activate a recorder you must
+    use the WBEM connection method:
+
+        `:class:`~pywbem.add_operation_recorder`
```

Comment:
Remove the middle back quote from the link above.

Reply:
fixed